src/openrct2/windows/guest_list.c

```diff
@@ -141,7 +141,7 @@ static sint32 _window_guest_list_highlighted_index; // 0x00F1EE10
 static sint32 _window_guest_list_selected_tab;      // 0x00F1EE12
 static sint32 _window_guest_list_selected_filter;   // 0x00F1EE06
 static sint32 _window_guest_list_selected_page;     // 0x00F1EE07
-static uint32 _window_guest_list_selected_view;  // 0x00F1EE13
+static uint32 _window_guest_list_selected_view;		// 0x00F1EE13
```

Comment:
Ok, I have an answer from IntelOrca:

> If it's C then I trust your opinion, it will be changed eventually

So if you could align the rest of this block (and the other referred block) with tabs, that should make it good to merge.